dist/assets/iframe.html

```diff
@@ -8,8 +8,11 @@
   </head>
   <body>
     <section data-main></section>
+
+    <!-- CRUFT: if you change these remember to update karma.conf.js:17-18 -->
     <script src="https://cdn.jsdelivr.net/g/lodash@2.4.2(lodash.underscore.min.js),handlebarsjs@1.3.0,jquery@2.2.4,momentjs@2.9.0,bootstrap@2.3.2"></script>
```

Comment:
Could store externals config in a hash somewhere and inject url into compiled html via webpack. Then could give karma same hash/config so there was single source of truth?